.version

```diff
@@ -1 +1 @@
-1.0.0
+1.0.1
```

Comment:
If this is a major version then this will increment to 2.0.0. That is only required if it is a breaking change. If the changes are not breaking, increment by a minor version.